endpoints/authentication/src/main.cpp

```diff
@@ -203,6 +203,9 @@ namespace irods::http::handler
 	IRODS_HTTP_API_ENDPOINT_ENTRY_FUNCTION_SIGNATURE(authentication)
 	{
 		if (_req.method() == boost::beast::http::verb::get) {
+			static const auto oidc_stanza_exists = irods::http::globals::configuration().contains(
+				nlohmann::json::json_pointer{"/http_server/authentication/oidc"});
```

Comment:
i think we've talked about renaming this stanza... itself, right?  "oidc" -> "openid_connect" ?

would be for that PR to update/change this code I believe...

Reply:
After some discussion, we've confirmed @MartinFlores751 has the changes for this in his fork. He'll be opening a PR for review soon.